src/components/Browse.tsx

```diff
@@ -126,9 +113,8 @@ const Browse = (props: any) => {
     if (!Array.isArray(newValue) || newValue.length !== 2) {
       return;
     }
-    console.log("old val", searchInputs.numberRange);
-    console.log("new val", newValue);
-    searchInputs.numberRange = newValue;
+    const newRange: number | number[] = newValue;
```

Comment:
Under what instances does newRange offer a number? Other than this part that I don't understand, I think everything else looks good. (If you don't mind, I'll be viewing replies after 7am tomorrow.)

Reply:
I was wondering this myself, but I dont actually know. I was following the material UI guide, in TypeScript, and their event seemed to have the possibility of coming as either type, so I left it as such. Perhaps I could make a check to disallow non array types.

Reply:
Ok sounds good! LGTM as well